packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6232,10 +6230,6 @@ export default class Meeting extends StatelessWebexPlugin {
     const LOG_HEADER = 'Meeting:index#addMedia -->';
     LoggerProxy.logger.info(`${LOG_HEADER} called with: ${JSON.stringify(options)}`);
 
-    if (this.meetingState !== FULL_STATE.ACTIVE) {
```

Comment:
addMedia() takes an `allowMediaInLobby` boolean in the options argument, so probably we should skip this check only if `allowMediaInLobby === true`

Reply:
good point, fixed